README.md

```diff
@@ -237,6 +237,11 @@ Use unbuffered output for the print statements. In C and C++ code, use
 
 Pipe the stderr output to file, e.g. `firefox 2> log`.
 
+If you are generating large amounts of data, piping your data through a fast 
+compressor could be convenient. `zstd` can process gigabytes per second, 
+while saving more than 90% on disk usage. Use `firefox 2>&1 >/dev/null | zstd 
```

Comment:
Nice suggestion, but I don't understand the `>/dev/null` here. Doesn't that throw away all the logging?

Reply:
This is suggested for piping stderr. It throws away the previous stdout as stderr becomes the new stdout.
https://stackoverflow.com/questions/2342826/how-can-i-pipe-stderr-and-not-stdout